app/View/Components/UserCard.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\View\Components;
+
+use Closure;
+use App\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class UserCard extends Component
+{
+    public $birthday;
+    /**
+     * Create a new component instance.
```

Comment:
add a public property $hasChirps which is set in the construct statement. the value should either be true or false, defaulting to false

you'll use this in the component blade